test/test_format.py

```diff
@@ -402,15 +403,16 @@ def setUpClass(cls):
                     cls.test_config[sub_dir.parts[-1]] = json.loads(f.read())
 
     def test_convert_times(self):
-        dt = datetime.datetime(2017, 1, 31,
-                               13, 22, 3)
+        dt = datetime.datetime(2017, 1, 31, 
+                               13, 22, 3, tzinfo=default_timezone())
```

Comment:
small note, i think its good to be explicit here, but passing this is fully optional, part of what this PR does is handle this transparently

the tests only need tzinfo added in returned results, all input dates will get it appended internally

in a sense this last commit adding all those tzinfos in optional places is now removing the testing for the functionality added by this PR

as i said i think it's good to be explicit in what is being tested, this comment is merely informative, not a request for a change

Reply:
Yeah it is an issue I think. 

With the naive datetimes having UTC tzinfo added, from memory the tests will fail if that's not your configured default unless you explicitly pass an aware object. So it is a bit of a patch job - we really need more detailed testing that checks those different scenarios.